CODEOWNERS

```
@@ -1,3 +1,4 @@
 * @carbon-design-system/ibm-cloud-cognitive-development
 
-/packages/security/ @dbrugger
+# Carbon for IBM Products > Security package
+/packages/security/ Jeff-Longshore  am-toussaint
```

I think it should actually be this.

```suggestion
/packages/security/ @jlongshore @amtoussam
```